script/openqa-clone-custom-git-refspec

```diff
@@ -159,6 +159,9 @@ Please try 'curl $json_url' or select another job, e.g. in the same scenario: $h
         productdir="${productdir/\/\//\/}" # avoid consecutive slashes
         local old_needledir
         old_needledir=$(echo "$json_data" | jq -r '.NEEDLES_DIR | select (.!=null)') || throw_json_error "$json_url" "$json_data"
+	if [[ -n $old_needledir ]]; then
```

Comment:
I would guess `$old_needledir` is always set (like we said, NEEDLES_DIR in vars.json is written by isotovideo). Why would you want to set NEEDLE_DIR depending on that, practically always?

Reply:
commit updated https://github.com/os-autoinst/openQA/commit/4704d764f4bc0f1ced1cdbd4d25b8acee3a6d372
https://openqa.opensuse.org/tests/4372387# 
https://openqa.opensuse.org/tests/4372388# < openqa-to-openqa